### ISSUE TITLE
Unable to send and receive image and audio from iPhone to android

### ISSUE THREAD

**Issue by jayarajb1116** [ChatSecure/ChatSecure-iOS#443]
I am an iOS developer. I am working on ChatSecure. When I send encryption request from iOS to android it sends some error 

2016-02-25 10:49:50:988 ChatSecure[5583:80975] SEND: <message type="chat" to="XXXX@xmpp1.mobile-crypto.com" id="053291CC-CA88-405B-B791-85FE635897F6"><body>?OTRv23?
&lt;b&gt;XXXX@xmpp1.mobile-crypto.com&lt;/b&gt; has requested an &lt;a href="https://otr.cypherpunks.ca/"&gt;Off-the-Record private conversation&lt;/a&gt;.  However, you do not have a plugin to support that.
See &lt;a href="https://otr.cypherpunks.ca/"&gt;https://otr.cypherpunks.ca/&lt;/a&gt; for more information.</body><active xmlns="http://jabber.org/protocol/chatstates"/><private xmlns="urn:xmpp:carbons:2"/><request xmlns="urn:xmpp:receipts"/></message>

And I am unable to send and receive image also. When I send image from Android to iOS there is a method being called in iOS. But I receive error message as follows

Error Domain=urn:ietf:params:xml:ns:xmpp-stanzas Code=406 "<error type="modify" code="406"><not-acceptable ....

Thanks in advance....

**Comment by chrisballinger** [comment]
The Android client must support OTR and OTRDATA.

**Comment by jayarajb1116** [comment]
**Note** The iOS device is using ChatSecure App and the Android device is using Conversation App.

**Comment by therob84** [comment]
I have the same problem!

Image/file sending with...
1.) ...from ChatSecure@iOS  (v3.1.3) --> Conversations@Android (v1.11.2): 
message "waiting" appears in ChatSecure --> no message appears at all at Conversations

2.) ...from Conversations@Android (v1.11.2) --> ChatSecure@iOS  (v3.1.3)
image is sent at Conversations and is displayed inline --> receiver at ChatSecure only gets an Link to the image/file (and no inline display)

Sending is done via OTR and both using the server jabber.de
Both clients can chat with each other with and without OTR.
Image sending between Conversations clients without any probem (not encrypted, OTR and OMEMO works all.)

Will you claim Conversations not to comply one of the XEP standards or can you find a problem/bug within ChatSecure?

**Comment by therob84** [comment]
**Update:**
Answer from Conversations Programmers in the conference-chat:

> "ChatSecure and Conversations are incompitible when it comes to file transfer"

Suggested solution is...

> "... to implement Jingle FT version 4, which should be the most compatible file transfer method currently" and ChatSecure-Team do "...already know about it / working on it."


Can you tell us somthing more, @chrisballinger ?

**Comment by mytbk** [comment]
So does ChatSecure-iOS supports http_upload as a file sharing method if the server supports it? I think this can be more interoperable.

**Comment by thundergreen** [comment]
Afaik nope.i have tried and it fails ... Some news for this from devs?

Thorsten Fröhlich

Am 18.07.2016 07:41 schrieb "mytbk" <notifications@github.com>:

**Comment by bertzi** [comment]
hi

the folowing methods are working for me with prosody server
--- unencrypted 
filetransfer from conversation/android to chatsecure/ios, ios client gets a url link
the other way doesent work because it is not implemented in chatsecure

--- encrypted ( OTR ), ( OMEMO not implemented in chatsecure, zom messenger will be probably the answer)
i didnt get this work in any direction.

did someone get filetransfer work from chatsecure to conversations ?

**Comment by chrisballinger** [comment]
File transfer between ChatSecure and Conversations is not yet supported,
but it's on our roadmap.

**Comment by n00b12345** [comment]
+1 to this.

**Comment by c-alpha** [comment]
+1 from me, too. 

Don't know how difficult it will be to fix, but w/o picture sending it'll be hard to convince Whatsappers & others to switch.

**Comment by 2Belette** [comment]
I agree qith @c-alpha it would be a huge help to move users to chatesecure

**Comment by chrisballinger** [comment]
It's almost done! Just putting on the finishing touches. Hopefully can put
a beta out by the end of next week, depending on how busy I am at WWDC.
Join the TestFlight group for beta access when its ready:
https://goo.gl/forms/Fp37cU5I8ZslO3Y42

**Comment by 2Belette** [comment]
@chrisballinger cool many thanks it is a wonderful news :) I am happy to see how fast Chatsecure is progressing on the "missing pieces" to shift users to XMPP/Chatsecure :)
